nmdc_runtime/site/graphs.py

```diff
@@ -146,3 +152,19 @@ def ingest_metadata_submission():
     database = translate_portal_submission_to_nmdc_schema_database(metadata_submission)
     run_id = submit_metadata_to_db(database)
     poll_for_run_completion(run_id)
+
+@graph
+def ingest_neon_metadata():
```

Comment:
I made a change to the submission portal ingestion pipeline that must have gone in after this branch was created because I don't see it here. But if you look in `main` I made two different graphs `translate_metadata_submission_to_nmdc_schema_database` -- which is like the "dry run" version, it just does the translation, validates it, and stores the output -- and `ingest_metadata_submission` -- which actually submits the translation result to Mongo. You might want to consider the same pattern here.

Reply:
Yes, for sure, that's a good pattern to follow, can choose to run either pipeline based on the action that the user wants to accomplishm I'll split out the "dry run" and actual submission to Mongo graphs into separate methods.

I'm curious though, this pattern hasn't been applied to the gold_translator graphs `gold_study_to_database()`. Do we want to add it to the GOLD translation layer as well?

Reply:
Re: GOLD, yeah probably, but no real urgency to do that.